lib/tools.js

```diff
@@ -98,24 +98,19 @@ function asJson (obj, msg, num, time) {
         value = serializers[key] ? serializers[key](value) : value
 
         switch (typeof value) {
-          case 'undefined':
-            break
-          case 'string':
-            value = (stringifiers[key] || asString)(value)
-            if (value !== undefined) {
-              data += ',"' + key + '":' + value
-            }
-            break
+          case 'undefined': continue
           case 'number':
           case 'boolean':
             data += ',"' + key + '":' + value
+            continue
```

Comment:
I was wondering if we should call the stringifiers for booleans and numbers as well. What do you think?

Reply:
yeah this is a good point, if there is a stringifier it may be a redactor 
and numbers definitely have a need to be redacted, and why not booleans as well,
particular in removal instead of redaction case

it might screw up inlining again though 🙈 I'll check

Reply:
@mcollina - updated and still inlines, also tests added